draft-lenders-core-coap-dtls-svcb.md

```diff
@@ -102,7 +102,103 @@ same manner as for any other service secured with transport layer security, as
 described in {{-svcb}}. Other authentication mechanisms are currently out of scope.
 
 # Constructing a CoAP URI from a SVCB Resource Record
```

Comment:
To avoid unintentional (re-)statement of URI construction: IMHO this section should be recontextualized or retitled:

```suggestion
# Constructing a CoAP Request from a SVCB Resource Record
```